### ISSUE TITLE
Dropdown `boundary` option not working as expected

### ISSUE THREAD

**Issue by jackmu95** [bootstrap-vue/bootstrap-vue#1819]
I'am not sure if its related to `bootstrap-vue` or `Bootstrap` itself, but I'am not able to get dropdowns working in a horizontally scrolling container with the `boundary` option set to `viewport`.

See: https://codesandbox.io/s/vv80yn9j07

- Operating system and version: Any
- Browser and version: Any
- Version of Bootstrap-Vue: 2.0.0-rc.9

**Comment by rinick** [comment]
it works fine when I try it on https://bootstrap-vue.js.org/play
maybe codesandbox has some css that conflicts with popper.js?

**Comment by jackmu95** [comment]
@rinick Have you applied the `overflow-x: auto;` styles to the `.dropdown-container`?
When I do that it isn't working in the playground too.

**Comment by rinick** [comment]
it's because of `.col` class has `position:relative` if you add `position: inherit;` to your dropdown container, the issue is fixed.  but I'm not sure what will be broken in bootstrap grid system when it's not relative

**Comment by jackmu95** [comment]
@rinick Thanks for the investigation.

I created an issue [over at Bootstrap](https://github.com/twbs/bootstrap/issues/26512).

**Comment by emanuelmutschlechner** [comment]
I think this relates somewhat to this https://github.com/twbs/bootstrap/issues/24251

The reason why `position: inherit`, i.e `position: static` does work can be found on [w3.org](https://www.w3.org/TR/CSS22/visudet.html#containing-block-details):
> If the element has 'position: absolute', the containing block is established by the nearest ancestor with a 'position' of 'absolute', 'relative' or 'fixed', in the following way:

> In the case that the ancestor is an inline element, the containing block is the bounding box around the padding boxes of the first and the last inline boxes generated for that element. In CSS 2.2, if the inline 
element is split across multiple lines, the containing block is undefined.

> Otherwise, the containing block is formed by the padding edge of the ancestor.
If there is no such ancestor, the containing block is the initial containing block.

In short:
The absolutely positioned element is positioned relative to its nearest positioned ancestor that is **not static**.
